### DISCUSSION TITLE
Working with enums

### DISCUSSION THREAD

**Discussion by ru-van-urk** [graphql-nexus/nexus#923] [Q&A]
Hi all,

First of all Nexus is working amazing for me, I really enjoy using it. Now I have an unique use case which I can not seen to figure out. I want to store the days of the week as an number in my postgresql database (0 = Sunday, 1= Monday, 2 = Tuesday etc.) thus I want to limit the options to store in the database. I thought of using an enum for this so I can assign the value to a number:

![image](https://user-images.githubusercontent.com/36665384/120331689-5476aa00-c2ee-11eb-976e-45a7a35476e3.png)

This works and I can use it but my resolver keeps saying:

![image](https://user-images.githubusercontent.com/36665384/120330941-981ce400-c2ed-11eb-9caa-22864f49c086.png)

I have been at it all day and it is driving me crazy, anyone a suggestion on how to tackle this.

Thanks in advance,

Ruben S.

Some pictures of my code:

![image](https://user-images.githubusercontent.com/36665384/120331796-7112e200-c2ee-11eb-82a6-e429437db698.png)

![image](https://user-images.githubusercontent.com/36665384/120331842-7c660d80-c2ee-11eb-9454-18f0111cd721.png)

**Comment by Stephane-Pires** [comment]
Hi, i'm having a similar issue right now, have you find a way to solve this ? 

@joxy379

**Reply by ru-van-urk** [reply]
Hi Stephanie, 

I never did find a solution to this problem. And I don’t use nexus anymore. I hope you will find a solution.

Cheers,

Ruben

**Reply by Stephane-Pires** [reply]
I found it.  

I had a mismatch between the types generated  (and handled)  by prisma and nexus. 

Thanks for your answer anyway

Stéphane
